apps/projects/app/components/Content/IssueDetail/EventsCard.js

```diff
@@ -28,13 +25,6 @@ const IssueEvent = ({ user, ...props }) => {
         </IssueEventAvatar>
         <IssueEventDetails>
           <Text.Block size="small">
-            <Link
-              href={user.url}
-              target="_blank"
-              style={{ textDecoration: 'none', color: `${theme.link}` }}
-            >
-              {user.login}
```

Comment:
I want to double-check that we want to remove the link to the user's profile on GitHub. That seems useful to me.

Reply:
The avatar should be clickable. Is that not the case?

Reply:
It wasn't the case before, it is now. One additional question then:

![Screenshot from 2019-11-27 14-54-59](https://user-images.githubusercontent.com/34452131/69729079-00519f00-1126-11ea-85a9-33d740ef562e.png)

In panels it's the old way - avatar photo and username, usename being the link. Adjust? Username as plain text, avatart photo as link?

Reply:
I realized there is more to be fixed in the panels. Needs separate ticket and PR. Created #1696 for it.